packages/devtools_app/lib/src/flutter/connect_screen.dart

```diff
@@ -127,34 +129,18 @@ class _ConnectScreenBodyState extends State<ConnectScreenBody> {
 
   Future<void> _connect([_]) async {
     final uri = normalizeVmServiceUri(controller.text);
-    bool firstError = true;
```

Comment:
I don't understand why this logic is not part of `FrameworkCore.initVmService`, so I've moved it there.

The code is set up such that only one error will be reported, and I'd think we would want to handle incorrect VM URIs the same regardless of Flutter or web.

Reply:
that is reasonable. Just go ahead and back in the early failure check for `if (uri == null || !uri.isAbsolute) `
which i think isn't in framework core.

Reply:
normalizeVmServiceUri is already handling the fragment part of the URI, so I've gone and made sure it's also making sure the URI has a scheme.  Test added too.